### PR TITLE
RavenDB-12540 Slow tests Server.Replication.ReplicationBasicTestsSlow…

### PR DESCRIPTION
**Pull request by efratshenhar**
….DisableExternalReplication

### CODE REVIEW

File: test/SlowTests/Server/Replication/ReplicationBasicTestsSlow.cs [hunk 1]
```diff
@@ -119,7 +121,17 @@ public async Task DisableExternalReplication()
                     Disabled = true
                 };
                 var res = await store1.Maintenance.SendAsync(new UpdateExternalReplicationOperation(external));
+                using (Server.ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+                {
+                    var value = WaitForValue(() =>
```
Comment (ayende) [inline test/SlowTests/Server/Replication/ReplicationBasicTestsSlow.cs]: Why do you have this change?

Comment (efratshenhar) [reply test/SlowTests/Server/Replication/ReplicationBasicTestsSlow.cs]: Wait for the topology to change

Comment (ppekrol) [reply test/SlowTests/Server/Replication/ReplicationBasicTestsSlow.cs]: @efratshenhar check this PR https://github.com/ravendb/ravendb/pull/7688
It looks like we did not applied that to 4.0

Comment (ayende) [reply test/SlowTests/Server/Replication/ReplicationBasicTestsSlow.cs]: I understand what it does, but why are you waiting for topology change when updating external replication?

**Pull request merged**